onnxruntime/contrib_ops/cuda/bert/cutlass_fmha/memory_efficient_attention.h

```diff
@@ -24,6 +24,7 @@ struct MemoryEfficientAttentionParams {
   int32_t max_sequence_length;
   int32_t qk_head_size;
   int32_t v_head_size;
+  int32_t local_window_size;
```

Comment:
Either set default value = -1;
or search MemoryEfficientAttentionParams, and change every place: https://github.com/search?q=repo%3Amicrosoft%2Fonnxruntime%20%20MemoryEfficientAttentionParams&type=code